connector/src/main/java/org/jboss/as/connector/subsystems/resourceadapters/RaOperationUtil.java

```diff
@@ -78,7 +77,6 @@
 import static org.jboss.as.connector.subsystems.resourceadapters.Constants.WM_SECURITY;
 import static org.jboss.as.connector.subsystems.resourceadapters.Constants.WM_SECURITY_DEFAULT_GROUPS;
```

Comment:
@tadamski Should the `WM_SECURITY_DEFAULT_GROUPS` be deprecated as well? And the `WM_SECURITY_MAPPING_*`  mappings below?

Reply:
@Skyllarr  hmm I was looking at those but my understanding is that this config could be used to map WildFly specific (Elytron included) security to resource adapter specific one? is it not the case?

Reply:
@tadamski you're right, these should not be deprecated